### PR TITLE
Add api for message history (MT-93)

### PR DESCRIPTION
**Pull request by approximator**

### CODE REVIEW

File: backend/config/settings.py [hunk 1]
```diff
@@ -292,6 +292,18 @@
...
+            "handlers": ["console"],
+        },
+        "register": {
+            "level": "DEBUG",
+            "handlers": ["console"],
+        },
+        "django.channels.server": {
+            "level": "INFO",
+            "handlers": ["console"],
+        }
```
[the remark this answers is not in the document]
Comment (approximator) [reply backend/config/settings.py]: Fixed

File: backend/config/urls.py [hunk 2]
```diff
@@ -47,10 +47,19 @@
...
+
+def get_file(file_name: str) -> str:
+    """Return the content of the file"""
+
+    examples_dir = BASE_DIR / "tests"
+    file_path = examples_dir / file_name
+    if not file_path.exists() or file_path.is_dir() or file_path.suffix != ".html":
+        return open(examples_dir / "login_example.html", "r", encoding="utf-8").read()
+
+    return open(file_path, "r", encoding="utf-8").read()
```
[the remark this answers is not in the document]
Comment (approximator) [reply backend/config/urls.py:65]: Fixed

File: backend/config/urls.py [hunk 3]
```diff
@@ -47,10 +47,19 @@
 if settings.DEBUG:
     urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
 
+
+def get_file(file_name: str) -> str:
+    """Return the content of the file"""
+
+    examples_dir = BASE_DIR / "tests"
+    file_path = examples_dir / file_name
+    if not file_path.exists() or file_path.is_dir() or file_path.suffix != ".html":
```
[the remark this answers is not in the document]
Comment (approximator) [reply backend/config/urls.py:62]: Fixed

**Review by oliabn — APPROVED** [review]

File: backend/config/urls.py [hunk 4]
```diff
@@ -47,10 +47,34 @@
...
+        not request_file_name.endswith(".html")
+        or request_file_name.count(".") > 1
+        or "/" in request_file_name
+        or "\\" in request_file_name
+    ):
+        return examples_dir / "login_example.html"
+
+    clean_file_name = request_file_name[:-5] + ".html"
+    file_path = examples_dir / clean_file_name
+    if not file_path.exists() or file_path.is_dir():
```
[the remark this answers is not in the document]
Comment (approximator) [reply backend/config/urls.py]: Fixed

[the remark this answers is not in the document]
Comment (approximator) [reply backend/config/urls.py]: Fixed

**Pull request merged**